master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+   gateway   *         130.211.10.121          80        1d
+   ```
+
+   The address of the ingress service would then be
+   
+   ```bash
+   export GATEWAY_URL=130.211.10.121:80
+   ```
+
+1. If your cluster does not support external load balancers, you can use the public IP of the worker
+node, along with the NodePort, to access the ingress. The IP & port can be obtained from the output 
+of the following command:
+   
+   ```bash
+   export GATEWAY_URL=$(kubectl get po -n istio-system -l istio=ingress -o 'jsonpath={.items[0].status.hostIP}'):$(kubectl get svc istio-ingress -n istio-system -o 'jsonpath={.spec.ports[0].nodePort}')
+   ```
+
+Point your browser to `http://$GATEWAY_URL/` to confirm the YAO Bank application is functioning 
+correctly.  It may take several minutes for all the services to come up and respond, during which
+time you may see 404 or 500 errors.
+
+### The need for policy
+
+Although Calico & Istio are running in the cluster, we have not defined any authorization policy.  
+Istio was configured to mutually authenticate traffic between the pods in your application, so only 
+connections with Istio-issued certificates are allowed, and all inter-pod traffic is encrypted with 
+TLS.  That's already a big step in the right direction.
+
+But, let's consider some deficiencies in this security architecture:
+
+ * All incoming connections from workloads in the Istio mesh are equally trusted
+ * Possession of a key & certificate pair is the *only* access credential considered.
+ 
+To understand why these might be a problem, let's take them one at a time.
+
+#### Trusting workloads
+
+Trusting connections from any workload in the Istio mesh is a poor security architecture because, 
+like Kubernetes, Istio is designed to host multiple applications.  Some of those applications may 
+not be as trusted as others.  They may be operated by different users or teams with wildly different
+security requirements.  We don't want our secure financial application microservices accessible from
+some hacky prototype another developer is cooking up.
+
+Even within our own application, the best practice is to limit access as much as possible.  Only 
+pods that need access to a service should get it.  Consider the YAO Bank application.  The customer
+web service does not need, and should not have direct access to the backend database.  The customer 
+web service needs to directly interact with clients outside the cluster, some of whom may be
+malicious.  Unfortunately, vulnerabilities in web applications are all too common.  For example, an
+[unpatched vulnerabiltiy in Apache Struts][struts cve] is what allowed attackers their initial
+access into the Equifax network where they then launched a devastating attack to steal millions of
+people's financial information.
+
+Imagine what would happen if an attacker were to gain control of the customer web pod in our
+application.
+
+Let's simulate this by `exec`'ing into that pod.
```

Comment:
It is OK...but a little informal: `exec`'ing. Consider instead "opening a bash shell inside that pod"